enigma-server/src/main/java/cuchaz/enigma/network/DedicatedEnigmaServer.java

```diff
@@ -96,27 +97,27 @@ public static void main(String[] args) {
 		int port = parsedArgs.valueOf(portOpt);
 		char[] password = parsedArgs.valueOf(passwordOpt).toCharArray();
 		if (password.length > EnigmaServer.MAX_PASSWORD_LENGTH) {
-			System.err.println("Password too long, must be at most " + EnigmaServer.MAX_PASSWORD_LENGTH + " characters");
+			Logger.error("Password too long, must be at most " + EnigmaServer.MAX_PASSWORD_LENGTH + " characters");
```

Comment:
This could be interpolation, but I think javac will constant-fold the number into the string at compile time?

Reply:
I'll switch to interpolation, just so no one sees concatenation here and uses it in new logging they add